### ISSUE TITLE
Issue with kibana_http/generate_certs.sh

### ISSUE THREAD

**Issue by Shahard2** [wazuh/wazuh-kubernetes#159]
After running it getting the next error:

Logs:

Generating a 2048 bit RSA private key
................................+++
..............................................................................................+++
writing new private key to 'key.pem'
-----
error, no objects specified in config file
problems making Certificate Request

**Comment by raul-gherman-modaoperandi** [comment]
encountered the same
my workaround was (this is the actual content of the file):

```
#!/bin/bash

DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
cd $DIR

openssl req -x509 -batch -nodes -days 365 -newkey rsa:2048 -keyout key.pem -out cert.pem  -subj "/C=US/L=California/O=Company/CN=kibana_http"
```

**Comment by bloodberrys** [comment]
I have the same problem with macOS big sur, my workaround is adding -subj "/" in OpenSSL command. 

```
openssl req -x509 -batch -nodes -days 365 -newkey rsa:2048 -keyout 'key.pem' -out 'cert.pem' -subj "/"
```

Ref: https://github.com/openssl/openssl/issues/3536#issuecomment-519937297

**Comment by NeckBeardPrince** [comment]
Mind-blowing that this is still an open issue after almost two years, and it's a deployment breaker.

**Comment by cyberresearch-us** [comment]
Yup...Just encountered it myself. However, I have watched some tutorials that do not get this issue.
